source/getting-started/concepts-terminology.markdown

```diff
@@ -1,70 +1,72 @@
+Integrations are pieces of software that allow Home Assistant to connect to other software and platforms. For example, a product by Philips called Hue would use the Philips Hue {% term integration %} and allow Home Assistant to talk to the hardware controller Hue Bridge. Any Home Assistant compatible {% term devices %} connected to the Hue Bridge would appear in Home Assistant as [devices](#devices--entities).
 
 ![Integrations](/images/getting-started/integrations-new.png)
 
-For a full list of compatible integrations, refer to the [integrations](/integrations) documentation.
+For a full list of compatible {% term integrations %}, refer to the [integrations](/integrations) documentation.
 
-Once an integration has been added, the hardware and/or data are represented in Home Assistant as [devices and entities](#devices--entities).
+Once an {% term integration %} has been added, the hardware and/or data are represented in Home Assistant as [devices and entities](#devices--entities).
 
 ## Entities
-{% term Entities %} are the base building block to hold data in Home Assistant. An {% term entity%} represents a {% term sensor %}, actor, or function in Home Assistant. {% term Entities %} are used to monitor physical properties or to control other entities. An entity is usually part of a {% term device %} or a {% term service %}. Entities have {% term states %}.
 
-## Devices 
+Entities are the basic building block to hold data in Home Assistant. An {% term entity %} represents a {% term sensor %}, actor, or function in Home Assistant. Entities are used to monitor physical properties or to control other {% term entities %}. An {% term entity %} is usually part of a {% term device %} or a {% term service %}. Entities have {% term states %}.
 
-Devices are a logical grouping for one or more entities. A {% term device %} may represent a physical device which can have one or more sensors. The sensors appear as entities associated to the device. For example, a motion sensor is represented as a device. It may provide motion detection, temperature, and light levels as entities. Entities have states such as *detected* when motion is detected and *clear* when there is no motion.
+## Devices
 
-![Home Assistant Device](/images/getting-started/home-assistant-device_01.png)
+Devices are a logical grouping for one or more {% term entities %}. A {% term device %} may represent a physical {% term device %} which can have one or more sensors. The sensors appear as entities associated to the {% term device %}. For example, a motion sensor is represented as a {% term device %}. It may provide motion detection, temperature, and light levels as {% term entities %}. Entities have states such as *detected* when motion is detected and *clear* when there is no motion.
```

Comment:
```suggestion
Devices are a logical grouping for one or more {% term entities %}. A {% term device %} may represent a physical {% term device %}, which can have one or more sensors. The sensors appear as entities associated with the {% term device %}. For example, a motion sensor is represented as a {% term device %}. It may provide motion detection, temperature, and light levels as {% term entities %}. Entities have states such as *detected* when motion is detected and *clear* when there is no motion.
```